File: app/src/main/java/com/example/facebookjava/Story.java
package com.example.facebookjava;

import java.util.UUID;

public class Story {
    private int profile;
    private String fullname;
    public boolean isAddStroyView = false;

    public Story(int profile, String fullname) {
        this.profile = profile;
        this.fullname = fullname;
    }

    public Story() {
        isAddStroyView = true;
    }

    public int getProfile() {
        return profile;
    }

    public void setProfile(int profile) {
        this.profile = profile;
    }

    public String getFullname() {
        return fullname;
    }

    public void setFullname(String fullname) {
        this.fullname = fullname;
    }

}

File: app/src/main/java/com/example/facebook/Story.kt
package com.example.facebook

class Story {
    var profile = 0
    var fullname: String? = null
    var isAddStroyView = false

    constructor(profile: Int, fullname: String?) {
        this.profile = profile
        this.fullname = fullname
    }

    constructor() {
        isAddStroyView = true
    }
}
